package.json

```diff
@@ -116,7 +116,7 @@
     "sift": "^3.2.7",
     "style-loader": "^0.16.1",
     "superagent": "^3.5.2",
-    "svgo": "^0.7.2",
+    "svg-url-loader": "^2.0.2",
```

Comment:
This loader is not optimising the loaded `svg`. We have to add [svgo loader](https://github.com/rpominov/svgo-loader) first.